streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImpl.java

```diff
@@ -863,45 +864,15 @@ public static void addTotalCountAndSumMetricsToSensor(final Sensor sensor,
         );
     }
 
-    public static void maybeMeasureLatency(final Runnable actionToMeasure,
```

Comment:
Moved these util functions to `AbstractTask` since we need them for both standby tasks and active tasks now.